app/value_objects/pdf_submission_detail.rb

```diff
@@ -0,0 +1,3 @@
+class PdfSubmissionDetail
+  def initialize(params = {}); end
+end
```

Comment:
noop, needs to be refactored out

Reply:
When? :)

Reply:
Would prefer to tackle the process_submission_service as an isolated task.
Will push for this before we modify this area of the code again.